lib/addons/src/index.js

```diff
@@ -2,7 +2,7 @@ export class AddonStore {
   constructor() {
     this.loaders = {};
     this.panels = {};
-    this.channel = null;
+    this.channel = { on() {}, emit() {} };
```

Comment:
This was giving us so much grief, I thought we could do without?

To the best of my abilities, everything actually works, there's no multiple loading going on.

It should never occur to not have set the channel.

Having said that, I think it's just really wrong to have a setup where side effects have such a major impact on whether a test can be run.

The channels is completely unneeded to run the snapshots for example, yet they have caused a lot of pain for a lot of users.

I just want people to be able to use and develop this. I think this code facilitates that better then without.